### DISCUSSION TITLE
DIV sizing difficulties

### DISCUSSION THREAD

**Discussion by fl8k** [quasarframework/quasar#17129] [General - Components / Directives / etc] [answered]
I am having some difficulty with my html/styling of some DIVs. Whenever I add a particular DIV inside another DIV, the height of the parent DIV expands unnecessarily. When I remove the inner DIV, the height is as expected/desired. Can anyone help me understand why adding the inner DIV is causing the height expansion of the parent? 

https://codepen.io/fl8k/pen/abxPoYV

![no_div](https://github.com/quasarframework/quasar/assets/66229473/240b551e-de80-40c1-8647-e6a41cecfbba)
![with_div](https://github.com/quasarframework/quasar/assets/66229473/f9b02e2b-7b95-4299-8f54-7a5ac2b4a9ea)

**Comment by hubdidub** [comment, marked as answer]
There are some typos in the html: You wrote `<div calss="col-1">` ("calss" instead of "class"). So the `col-1` doesn't actually apply there. 
When you correct it, then it works as you wanted.

I can't really explain _why_, though.  
My guess is that inside a `row`, having a mixture of `col-N` and non-`col` children isn't allowed and is causing some weird behavior.

**Reply by fl8k** [reply]
Wow that's embarrassing.. that solves it. Thanks
